packages/react/src/components/SuggestionMenu/SuggestionMenuController.tsx

```diff
@@ -66,12 +68,42 @@ export function SuggestionMenuController<
     );
   }, [editor, onItemClick]);
 
+  const insertSlashMenuOption = (editor: any) => ({
+    title: "Emoji",
+    //we are simulating a : keydown event to trigger the emoji menu
+    onItemClick: () => {
+      const keyboardEvent = new KeyboardEvent('keydown', {
+        key: ':',          // key identifier (DOMString), optional
+        code: 'Colon',     // key code identifier (DOMString), optional
+        keyCode: 186,      // key code value (unsigned long), optional
+        which: 186,        // legacy keyCode, optional
+        charCode: 0,       // character code value (unsigned long), optional
+        bubbles: false,    // bubbles flag (boolean), optional
+        cancelable: false, // cancelable flag (boolean), optional
+        composed: false,   // composed flag (boolean), optional
+        ctrlKey: false,    // control key flag (boolean), optional
+        altKey: false,     // alt key flag (boolean), optional
+        shiftKey: false,   // shift key flag (boolean), optional
+        metaKey: false     // meta key flag (boolean), optional
+      });
+      editor.domElement.dispatchEvent(keyboardEvent);
+    },
+    aliases: [
+      "emoji",
+      "emote",
+      "face",
+    ],
+    group: "Other",
+    icon: <MdEmojiEmotions />,
+  });
+
   const getItemsOrDefault = useMemo(() => {
     return (
       getItems ||
       ((async (query: string) =>
         filterSuggestionItems(
-          getDefaultReactSlashMenuItems(editor),
+          //STEP 6: SHOW THE EMOJI OPTION IN THE SLASH MENU
```

Comment:
You should move the emoji slash menu item to `getDefaultSlashMenuItems`/`getDefaultReactSlashMenuItems`. Should be able to copy the same structure as the existing items.